scripts/linters/html_linter.py

```diff
@@ -358,6 +358,28 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.html_filepaths
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from htmllint error messages.
+
+        Args:
+            lint_messages: str. Messages returned by the html linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # Extracting messages by removing number of files linted and number of
+        # error messages from the end of message. Becuase we last two lines
```

Comment:
"Because we last two lines like this" does not make sense as an English sentence. Please reword.